packages/webapp/src/components/ui/RequestPanelBody/index.tsx

```diff
@@ -119,6 +119,9 @@ export const RequestPanelBody: StandardFC<RequestPanelBodyProps> = memo(function
 				</Row>
 
 				{/* Request description */}
+				<div className='d-inline-block'>
+					<strong>{t('viewRequest.body.description')}</strong>
```

Comment:
I was going to chastise you for using `<strong>`...
but alas, I see that's what they are using in the rest of the component.